spec/factories/user.rb

```diff
@@ -2,8 +2,8 @@
   factory :user do
     username { SecureRandom.hex(3) }
     state 'activated' # otherwise the default from DB will be to 'temp'
-    first_name Faker::Name.first_name
-    last_name  Faker::Name.last_name
+    first_name { Faker::Name.first_name }
+    last_name  { Faker::Name.last_name  }
     trait :admin do
```

Comment:
this was a silly bug that was throwing off the specs.  User's were being created all with the same name, and the routine was calling `.uniq` on the names, making it look like there was only a single user with that email.